build-source-tarball.sh

```diff
@@ -137,6 +141,18 @@ do
     cp $built_package $TARBALL_ROOT/prebuilt/source-built/
 done
 
+echo 'Removing reference-only packages from tarball prebuilts...'
+
+for ref_package in $(find $SCRIPT_ROOT/reference-packages/packages-to-delete/ -name '*.nupkg' | tr '[:upper:]' '[:lower:]')
+do
+    if [ -e $TARBALL_ROOT/prebuilt/nuget-packages/$(basename $ref_package) ]; then
+        rm $TARBALL_ROOT/prebuilt/nuget-packages/$(basename $ref_package)
+    fi
+    if [ -e $TARBALL_ROOT/prebuilt/smoke-test-packages/$(basename $ref_package) ]; then
+        rm $TARBALL_ROOT/prebuilt/smoke-test-packages/$(basename $ref_package)
```

Comment:
I don't think this is right--these aren't included in the tarball that we give Red Hat, and it's only a way to transport packages that aren't available from any online source at that point for testing purposes. They're actually not included in the tarball.

Core problem is that they're called "prebuilts", come to think of it, when they aren't really.